### PR TITLE
#273, updated english text

### PR DESCRIPTION
**Pull request by elma13**
English translation, some details have changed in last pull request, this are minor details

### CODE REVIEW

File: lang/translation.json [hunk 1]
```diff
@@ -73,23 +73,23 @@
 	  },
 	  {
 	    "tag" : "word_maximum_value",
-	    "en" : "Filter on the maximal value",
+	    "en" : "Filter on the maximum value",
 	    "nl" : "Filteren op maximale waarde"},
 	    {
 	      "tag" : "tool_maximum_value_1_expl",
-	      "en" : "",
+	      "en" : "Every measurment reported by the sensor can provide information about the air quality at its location. However, if the sensor is dirty or damaged, then the measurment may not be representative of the air quality. This will often result in high measurment, for example 300 μg/m³. For the analysis you want to estimate the air quality, without distortions due to such outliers. To that end you can use a filter to remove measurements above a maximum value. By default, this filter is set to 10% percentile of the sensor measurements. You can adjust this filter yourself, so you can retain high measurement if needed. Set the maximum value to 1000 μg/m³, for instance, and look at the sensors one at a time.",
```
Comment (jspijker) [inline lang/translation.json:80]: spelling error in measurement, use 90-percentile

File: lang/translation.json [hunk 2]
```diff
@@ -421,12 +421,12 @@
         "nl": "Individuele tijdreeks"
       },
       {"tag" : "expl_indu_timeplot",
-        "en": "Here you can view the timeseries of the individual sensor.",
+        "en": "Here you can view the time series of the individual sensor. Additionally you can set the maximum value; measurments above this maximum will not be shown. Often when a sensor reports very high measurments, this is due to a dirty of faulty sensor. These ourliers are therefore not representative of the air quality, and are better left out of the analysis.",
```
Comment (jspijker) [inline lang/translation.json:424]: spelling errors: measurements (2x), outliers

File: lang/translation.json [hunk 3]
```diff
@@ -421,12 +421,12 @@
...
+        "en": "Click on a sensor on the map to view the graph of that sensor. The colored band indicates the confidence interval; more information about this confidence interval can found on the 'Information about the tool' tab. About the maximum value: sometimes a sensor reports very high measurments, for example 300 μg/m³, but that is not the true concentration in the air. Often this is because the sensor is faulty or dirty. In this situation you do not want to use these outliers in the analsysis, because they are nor representative of the air quality. For more information, see the 'Information about the tool' tab. ",
```
Comment (jspijker) [inline lang/translation.json:428]: this confidence interval can *be* found?, again spelling error measurements,

File: lang/translation.json [hunk 4]
```diff
@@ -445,12 +445,12 @@
         "nl": "Concentratieroos (%)"
       },
       {"tag" : "expl_concplot%",
-        "en": "",
+        "en": "To get a sensor of where the particulate matter comes from, the measurment concentrations are combined with wind data from the KNMI.",
```
Comment (jspijker) [inline lang/translation.json:448]: measurement again

**Comment by jspijker** [comment]
If I change to English, the text in the information popups (the 'I' button) is displayed in Dutch. This should be English, right?

**Review by jspijker — CHANGES_REQUESTED** [review]

File: lang/translation.json (hunk 1 above)
Comment (elma13) [reply lang/translation.json:80]: adjusted

File: lang/translation.json (hunk 2 above)
Comment (elma13) [reply lang/translation.json:424]: adjusted

File: lang/translation.json (hunk 4 above)
Comment (elma13) [reply lang/translation.json:448]: adjusted

**Pull request merged**